### ISSUE TITLE
contao-privacy für Contao 3.5?

### ISSUE THREAD

**Issue by Webstylerin** [friends-of-contao/contao-privacy#43]
Hallo zusammen,

soll die Erweiterung unter Contao 3.5 laufen? Hört sich zumindest für mich so an: "This extension provides some privacy features to Contao 3.5.x and 4.4.x (4.5.x)."

Via Paketverwaltung finde ich keine stabile Version und die Beta funktioniert nicht. (Statt Splash-Image oder No-Cookie für YT wird gar nichts im FE ausgeben.)

LG
Nicole

![bildschirmfoto 2018-10-19 um 09 13 17](https://user-images.githubusercontent.com/3033160/47203008-4105b300-d37f-11e8-980f-4c1b7c9d67ba.png)

**Comment by fritzmg** [comment]
Ja, die Version 1.x ist für Contao 3. Allerdings wurden dort evt. noch nicht alle Features installiert oder nachgezogen.

**Comment by Webstylerin** [comment]
Sollte dort YT funktionieren? Wie gesagt, sobald die Erweiterung aktiv ist, wird mit dem YT-Element rein gar nichts mehr im FE ausgegeben.

**Comment by fritzmg** [comment]
Ich denke das wurde 1:1 übernommen, da müssten aber noch Anpassungen für Contao 3 gemacht werden.
Wie es mit der contao-privacy Extension weiter geht, muss aber ohnhin noch entschieden werden.
